samples/stored-procedures/bulkImport.js

```diff
@@ -1,59 +1,59 @@
-﻿/**

```

Comment:
Is this file getting modified? Looks like it's not. Can we remove if it from the change?